### DISCUSSION TITLE
Best practices

### DISCUSSION THREAD

**Discussion by grepsuzette** [gnolang/gno#1683] [Q&A] [answered]
Sorry to open an issue for that. 
There is I think a doc by @moul to show the best practices for Gno (modeled after the famous one for Go).

Where is the address again?

**Comment by thehowl** [comment, marked as answer]
@grepsuzette Hey! Moving this to Discussions as we want to go back to using this more, I'm just lagging behind on creating the pointers to asking questions here because of the usual plethora of stuff going on :)

You may be interested in reading this issue --> https://github.com/gnolang/meetings/issues/38 it tries to tackle some problems that you've brought up yourself some time ago and explains a bit more what we intend to do with discussions. Feel free to ask anything here, and/or to pitch in on that issue itself...

Coming to the best practices at hand, here is the doc: https://docs.gno.land/concepts/effective-gno :clinking_glasses:

**Reply by grepsuzette** [reply]
Thank you, that's perfect!
